### ISSUE TITLE
SGE cluster system setup?

### ISSUE THREAD

**Issue by fultonardo** [TranslationalBioinformaticsIGTP/CNVbenchmarkeR#3]
Greetings!

I am experiencing issues setting up an SGE cluster system on Ubuntu 18.04 and Ubuntu 20.04 machines.  I am able to install and get the sge_qmaster running, but can't get past an error: "reresolve hostname failed: can't resolve host name
unable to send message to qmaster using port 6444 on host "/var/lib/gridengine": got unexpected parameters."  

I am wondering if you could recommend a setup for running CNVbenchmarkeR and an SGE cluster system, and provide any resources for assembly and setup of a standalone SGE cluster system.

Thank you, 
-Matt

**Comment by jpuntomarcos** [comment]
Sorry, I cannot give you any feedback because it was our sysadmin who set up the SGE system.

Regards,
J.

**Comment by fultonardo** [comment]
Okay!  Thanks again José!  

With the use of virtual machines I was able to get the CNVbenchmarkeR optimizer tools up and running with SGE in Ubuntu 16.04 as follows for DECoN and panelcn.MOPS:
DECoN: Ubuntu 16.04- R v3.4.4- DECoN v1.0.2- SGE- CNVBenchmarkeR
panelcn.MOPS: Ubuntu 16.04- R v.4.0.3- panelcn.MOPS v1.12.0- SGE- CNVbenchmarkeR

On Ubuntu 16.04, after I had R, DECoN/panelcn.MOPS, and CNVbenchmarkeR installed, I used the following link to install SGE:
https://gist.github.com/asadharis/9d14da97d9ad1f8eccc36dc14390e4e0

After installing SGE and making sure it was configured properly the CNVbenchmarkeR optimizer was able to run!  Once I have all of the datasets ready for testing I will move these builds to AWS EC2 servers to save computing time.

Cheers!
-Matt
